### PR TITLE
chore: Merge tasks into one extension

### PR DESCRIPTION
**Pull request by vdmgolub**
In marketplace: https://marketplace.visualstudio.com/items?itemName=Infracost.infracost-tasks

### CODE REVIEW

File: overview.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+This task does not set any direct outputs.
+
+## InfracostComment Task
+
+This Azure DevOps task takes `infracost breakdown` JSON output and posts it as a Azure DevOps Repos or GitHub comment. It assumes the `infracost` binary has already been installed using the *InfracostSetup* task. It uses the Azure DevOps access token or GitHub token, to post comments (you can override it with inputs). This task uses the [compost](https://github.com/infracost/compost) CLI tool internally.
+
+### Usage
+
+The action can be used as follows. See the [top-level readme](https://github.com/infracost/infracost-azure-devops) for examples of how this task can be combined with the setup task to run Infracost.
```
Comment (vdmgolub) [inline overview.md:53]: We may want to embed the examples/README.md links here so they will be visible on Marketplace page right away.

File: vss-extension.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  "id": "infracost-tasks",
+  "name": "Infracost Tasks",
+  "version": "0.0.1",
+  "publisher": "Infracost",
+  "targets": [
+    {
+      "id": "Microsoft.VisualStudio.Services"
+    }
+  ],
+  "description": "Tasks for setting up Infracost in your Azure DevOps pipeline and showing cloud cost estimates for Terraform in pull requests.",
```
Comment (vdmgolub) [inline vss-extension.json:12]: Seems like this is a proper name for the service, need to stick with it everywhere.
```suggestion
  "description": "Tasks for setting up Infracost in your Azure Pipelines and showing cloud cost estimates for Terraform in pull requests.",
```

**Comment by hugorut** [comment]
Nice, have you managed to test this on a live pipeline? From the looks of it the extension hasn't been installed on the pipeline, so I'm worried that any manual/automated tests you've done will just be from the old extensions.  

Screenshot 2022-01-13 at 10 05 31

(the new extension is shared with our org but not installed)

File: vss-extension.json (hunk 2 above)
Comment (aliscott) [reply vss-extension.json:12]: 👍  lgtm

**Comment by vdmgolub** [comment]
@hugorut I'm in the process right now. Weird thing I can't request Admin to install the extension.

**Pull request merged**